src/components/menu/MenuSharedTypes.ts

```diff
@@ -0,0 +1,12 @@
+// (C) 2007-2018 GoodData Corporation
+import { IDirection } from './Position/position';
+
+export type OpenAction = 'click' | 'hover';
+
+export type Direction = IDirection;
```

Comment:
why is the type renamed here?

Reply:
Resolved in previous conversation
https://github.com/gooddata/gooddata-react-components/pull/669#pullrequestreview-178750641